src/libs/actions/ReimbursementAccount/validateBankAccount.js

```diff
@@ -20,8 +20,6 @@ function validateBankAccount(bankAccountID, validateCode) {
                 DeprecatedAPI.User_IsUsingExpensifyCard()
                     .then(({isUsingExpensifyCard}) => {
                         const reimbursementAccount = {
-                            loading: false,
```

Comment:
I think we should replace `loading: false` with the new `isLoading: false`, no?

By adding 

```js
Onyx.merge(ONYXKEYS.FORMS.VALIDATION_STEP_FORM, {isLoading: false});
```

right after:

https://github.com/Expensify/App/blob/23ebd43804fc930cd640ef49a7fa874f8928d0bd/src/libs/actions/ReimbursementAccount/validateBankAccount.js#L26-L28